docs/docs-content/automation/palette-cli/commands/vmo.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_label: "VMO"
+title: "VMO"
+description: "Reference resource for the vmo command."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["palette-cli"]
+---
+
+Use the `vmo` command to migrate Virtual Machines (VMs) and import and deploy vSphere Open Virtual Appliances (OVAs).
+The VMs can then be used with the Virtual Machine Orchestrator (VMO).
+
+## Subcommands
+
+- [`deploy-ova`](#deploy-ova) - Deploy an imported vSphere OVA. This command requires you to have an OVA deployment
+  file. If you do not have one, you can generate an OVA with the `import-ova` subcommand.
+
+- [`import-ova`](#import-ova) - Import a vSphere OVA. This subcommand will generate an OVA deployment configuration
+  file. The configuration can then be deployed using the `deploy-ova` subcommand.
+
+- [`migrate-vm`](#migrate-vm) - Migrate one or more VMware vSphere VMs to Palette VMO.
+
+## Prerequisites
+
+- A Healthy VMO cluster. Refer to the [Create a VMO Profile](../../../vm-management/create-vmo-profile.md) for further
+  guidance.
+- One or more VMs hosted in VMware vSphere. The VMs should also operate one the
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html).
+
+## Limitations
+
+- You can only use the `vmo` subcommand with VMs hosted in VMware vSphere.
+
+## Deploy OVA
+
+Use the `deploy-ova` subcommand to deploy an imported vSphere OVA to Palette VMO. The following flags are supported by
+the `deploy-ova` subcommand.
```

Comment:
Additional context from the CLI we may want to include:

```
The OVA will be converted to QCOW2 format and an OVA deployment configuration file will be generated.

The QCOW2 image can either be directly uploaded to a DataVolume in K8s or converted to a Docker image
image and uploaded to a registry of your choosing.

Subsequently, the OVA can be deployed to K8s by invoking the deploy-ova command and providing the
configuration file generated during the import process.
```

Reply:
Reworded this and added it as a bit of context